projects/systelab-translate/README.md

```diff
@@ -8,6 +8,19 @@ Library with I18N tools to speed up our Angular developments
 npm install systelab-translate --save
 ```
 
+## Importing the library 
```

Comment:
This is part of How to use the library. It is a sub-case as it is not mandatory.